src/sentry/api/bases/organization_events.py

```diff
@@ -88,6 +88,22 @@ def get_snuba_query_args_v2(self, request, organization, params):
 
             snuba_args['selected_columns'] = fields
 
+        conditions = snuba_args.get('conditions')
+        # Add special fields to aggregations if missing
+        if conditions:
+            for condition in conditions:
+                field = condition[0]
+                if isinstance(field, (list, tuple)):
+                    continue
+                if field in SPECIAL_FIELDS:
+                    aggregation_included = False
+                    for aggregate in aggregations:
+                        if aggregate[2] == field:
```

Comment:
It looks like all the special fields have aggregations that are distinct. Do we need to handle the scenario where an aggregation has been added more than once?

Reply:
So here I'm going through the conditions. I imagine that the user could write something like:
`event_count:>1 event_count:<10` that's what I was thinking about when I wrote this. Let me know if you think differently there @markstory